src/v2/Apps/Show/Components/ShowInfoLocation.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react"
+import { Text } from "@artsy/palette"
+import { createFragmentContainer, graphql } from "react-relay"
+import { ShowInfoLocation_show } from "v2/__generated__/ShowInfoLocation_show.graphql"
+
+interface ShowInfoLocationProps {
+  show: ShowInfoLocation_show
+}
+
+export const ShowInfoLocation: React.FC<ShowInfoLocationProps> = ({ show }) => {
+  // Merge show location and fair location into a single object, favoring show
+  const location = Object.entries(show.location).reduce((acc, [key, value]) => {
+    return { ...acc, [key]: value ?? show.fair?.location?.[key] }
+  }, {} as typeof show.location)
+
+  const lines = [
+    location.display,
+    location.address,
+    location.address2,
+    [location.city, location.state, location.country]
+      .filter(Boolean)
+      .join(", "),
+    location.summary,
```

Comment:
`summary` on the other hand includes information denoting a fair as 'online only', for instance